AIAgent/launch_servers.py

```diff
@@ -86,7 +86,10 @@ def run_server_instance(port: int, start_server: bool) -> ServerInstanceInfo:
     launch_server = [
         "dotnet",
         "VSharp.ML.GameServer.Runner.dll",
-        "--checkactualcoverage",
+        "--datasetdescription",
```

Comment:
This argument is not required in server mode.

Reply:
Without this argument server doesn't launch:
```
Unhandled exception. System.ArgumentException: The value cannot be an empty string. (Parameter 'path')
   at System.ArgumentException.ThrowNullOrEmptyException(String argument, String paramName)
   at System.IO.File.ReadAllText(String path, Encoding encoding)
   at Main.loadGameMaps(String datasetDescriptionFilePath) in /Users/emax/Data/PySymGym/GameServers/VSharp/VSharp.ML.GameServer.Runner/Main.fs:line 42
   at Main.main(String[] args) in /Users/emax/Data/PySymGym/GameServers/VSharp/VSharp.ML.GameServer.Runner/Main.fs:line 252
```
moreover, this path should be valid:
```
Unhandled exception. System.IO.DirectoryNotFoundException: Could not find a part of the path '/Users/emax/Data/PySymGym/GameServers/VSharp/VSharp.ML.GameServer.Runner/bin/Debug/net7.0/maps/DotNet/Maps/dataset.json'.
   at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirError)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode, Func`4 createOpenException)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, UnixFileMode openPermissions, Int64& fileLength, UnixFileMode& filePermissions, Func`4 createOpenException)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.StreamReader.ValidateArgsAndOpenPath(String path, Encoding encoding, Int32 bufferSize)
   at System.IO.File.ReadAllText(String path, Encoding encoding)
   at Main.loadGameMaps(String datasetDescriptionFilePath) in /Users/emax/Data/PySymGym/GameServers/VSharp/VSharp.ML.GameServer.Runner/Main.fs:line 42
   at Main.main(String[] args) in /Users/emax/Data/PySymGym/GameServers/VSharp/VSharp.ML.GameServer.Runner/Main.fs:line 252
```

Reply:
This option is removed in latest commit